extensions/sql-assessment/src/tabs/assessmentMainTab.ts

```diff
@@ -107,18 +112,20 @@ export class SqlAssessmentMainTab extends SqlAssessmentTab {
 				light: this.extensionContext.asAbsolutePath('resources/light/database.svg')
 			};
 		const iconSize: number = 16;
+		const btnHeight: string = '26px';
 
 		const btnInvokeAssessment = view.modelBuilder.button()
 			.withProperties<azdata.ButtonProperties>({
-				label: this.invokeAssessmentLabel,
+				label: this.limitLongName(this.invokeAssessmentLabel),
```

Comment:
Why is this necessary? This string will never be over 40 chars

Reply:
we need to handle cases for very-very long database names and small screens (like 13 inch). The label is composed of "Invoke assessment for <databasename>", so limit it by length seems reasonable to solve the issue.

Reply:
Oh I didn't see you overwrote the title if it's a server connection....could you attach an image of what this looks like?

Reply:
(for future reference please always attach screenshots/gifs if you're making UI changes so reviewers can easily see what the changes look like)

Reply:
I have attached the screen with an issue at the level up comments.

Reply:
Why do you even need the database name here? The dashboard already indicates which database it is so my opinion is just that you should remove the name completely.